drafts/25-xxx-specifications.txt

```diff
@@ -296,6 +296,11 @@ di07. The non-directive directive
 7 Predefined macros
 ===================
 
+pm00. Macros beginning with a leading underscore followed by an uppercase letter
+      or a second underscore are reserved to the implementation. A program unit
+      shall not define a macro beginning with an underscore followed by an
+      uppercase letter or a second underscore.
```

Comment:
This is repetitive with the previous sentence, and does not cover `#undef`. 
Finally, "the implementation" does not appear to be formally defined in Fortran. I think we mean "the processor".

How about something more like:

```suggestion
      or a second underscore are reserved to the processor. Such reserved
      names shall not be the subject of a #define or #undef preprocessing directive.
```

Reply:
With this change, this rule becomes partially redundant with pm12.

We should probably strip down pm12 to only cover the `defined` identifier